config/src/test/kotlin/org/springframework/security/config/annotation/web/AuthorizeHttpRequestsDslTests.kt

```diff
@@ -961,4 +963,45 @@ class AuthorizeHttpRequestsDslTests {
         }
 
     }
+
+    @Test
+    fun `request when fully authenticated configured then responds ok`() {
```

Comment:
Will you please add another test that confirms that when a security context is not fully authenticated, then Security correctly forbids? For example, using `RememberMeAuthenticationToken` to simulate a remembered authentication.